satellite/satellitedb/storagenodeaccounting.go

```diff
@@ -219,25 +219,29 @@ func (db *StoragenodeAccounting) QueryStorageNodeUsage(ctx context.Context, node
 
 	start, end = start.UTC(), end.UTC()
 
-	query := `WITH r AS (
-			SELECT at_rest_total, start_time
-			FROM accounting_rollups
-			WHERE node_id = ?
-			AND ? <= start_time AND start_time <= ?
-		)
-		SELECT at_rest_total, start_time from r
+	query := `
+		SELECT at_rest_total, start_time::date
+		FROM accounting_rollups
+		WHERE node_id = $1
+		AND $2 <= start_time AND start_time <= $3
 		UNION
-		SELECT SUM(data_total) AS at_rest_total, DATE(interval_end_time) AS start_time
-			FROM storagenode_storage_tallies
-			WHERE node_id = ?
-			AND NOT EXISTS (SELECT 1 FROM r WHERE DATE(r.start_time) = DATE(interval_end_time))
-			AND (SELECT value FROM accounting_timestamps WHERE name = ?) < interval_end_time AND interval_end_time <= ?
-			GROUP BY start_time
-		ORDER BY start_time`
+		SELECT SUM(data_total) AS at_rest_total, interval_end_time::date AS start_time
+				FROM storagenode_storage_tallies
+				WHERE node_id = $1
+				AND NOT EXISTS (SELECT 1 FROM (
```

Comment:
I think we could combine these two nested selects, now that we've taken apart the CTE anyway, and get

```sql
    AND NOT EXISTS (
        SELECT 1 FROM accounting_rollups
        WHERE node_id = $1
        AND $2 <= start_time AND start_time <= $3
        AND start_time::date = interval_end_time::date
    )
```

Reply:
let me make this change and confirm all still looks good

Reply:
@JessicaGreben You are bad at PTO

Reply:
lol I have a free couple of mins.